schema.prisma

```diff
@@ -184,12 +184,15 @@ model Campaign {
 }
 
 model CampaignFile {
-  id            String   @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
-  filename      String   @db.VarChar(200)
-  campaignId    String   @map("campaign_id") @db.Uuid
-  campaign      Campaign @relation(fields: [campaignId], references: [id])
-  uploadedById  String   @map("uploaded_by_id") @db.Uuid
-  uploadedBy    Person   @relation(fields: [uploadedById], references: [id])
+  id           String   @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
+  filename     String   @db.VarChar(200)
+  role         String   @db.VarChar(200)
+  type         String   @db.VarChar(200)
```

Comment:
We better have two enum records for each of the `role` and `type` dimensions

```prisma
enum FileRole {
  ...
  @@map("file_role")
}

Reply:
I agree